Procfile

```diff
@@ -1,4 +1,4 @@
-web: gunicorn kuma.wsgi -b 0.0.0.0:8000
+web: gunicorn kuma.wsgi -w 2 -b 0.0.0.0:8000
```

Comment:
So this must have been an issue since we moved to gunicorn, not long ago. The runserver we used before is threaded and just spins up another thread to process incoming requests.